### PR TITLE
Fix: remove broken links from Security page

### PR DESCRIPTION
**Pull request by claudiob**
The list of people who has found security flaws includes links to websites that are no longer accessible.

This commit removes the broken links while maintaining the names in the list.

### CODE REVIEW

File: security/response/index.md [hunk 1]
```diff
@@ -39,37 +39,37 @@ We respect the time and talent that drives new discoveries in web security techn
 * [John Firebaugh](http://jfire.io)
 * [Kamil Sevi](https://twitter.com/kamilsevi)
 * [Marko Karppinen](http://www.karppinen.fi)
-* [Matasano Security](http://www.matasano.com)
+* Matasano Security
```
Comment (claudiob) [inline security/response/index.md:42]: Snap 2020-08-06 at 2 06 54 PM

File: security/response/index.md [hunk 2]
```diff
@@ -39,37 +39,37 @@ We respect the time and talent that drives new discoveries in web security techn
...
 * [nGenuity Information Services](http://www.ngenuity-is.com)
 * [ONZRA](http://www.onzra.com)
 * [Óscar Repáraz](http://www.reparaz.net)
 * [Rakan Alotaibi @hxteam](https://twitter.com/hxteam)
-* [Simon Brown](http://www.isimonbrown.co.uk)
-* [Tim Bach](https://twitter.com/tbach0729)
-* [Jan Habermann](https://twitter.com/habermann24)
+* Simon Brown
+* Tim Bach
+* Jan Habermann
```
Comment (claudiob) [inline security/response/index.md:51]: Snap 2020-08-06 at 2 07 01 PM
Snap 2020-08-06 at 2 07 12 PM
Snap 2020-08-06 at 2 07 20 PM

File: security/response/index.md [hunk 3]
```diff
@@ -39,37 +39,37 @@ We respect the time and talent that drives new discoveries in web security techn
...
 * [John Menerick](http://securesql.info)
-* [Prajal Kulkarni](http://prajalkulkarni.blogspot.com)
-* [Ajay Singh Negi](http://www.computersecuritywithethicalhacking.blogspot.in)
-* [Harsha Vardhan Boppana (Login Security Solution (P) Limited)](http://www.harshavardhan.me)
+* Prajal Kulkarni
+* Ajay Singh Negi
+* Harsha Vardhan Boppana (Login Security Solution (P) Limited)
```
Comment (claudiob) [inline security/response/index.md:55]: Snap 2020-08-06 at 2 07 29 PM
Snap 2020-08-06 at 2 07 35 PM
Snap 2020-08-06 at 2 07 38 PM

File: security/response/index.md [hunk 4]
```diff
@@ -39,37 +39,37 @@ We respect the time and talent that drives new discoveries in web security techn
...
 * [Frans Rosén](https://www.detectify.com)
 * [Rafay Baloch](http://rafayhackingarticles.net)
 * [M.R. Vignesh Kumar](https://twitter.com/vigneshkumarmr)
 * [Himanshu Kumar Das](https://twitter.com/mehimansu)
 * [Krutarth Shukla](https://twitter.com/KrutarthShukla)
 * [Ahmad Ashraff](https://twitter.com/yappare)
-* [InverseKey](http://inversekey.com)
+* InverseKey
```
Comment (claudiob) [inline security/response/index.md:62]: Snap 2020-08-06 at 2 07 42 PM

File: security/response/index.md [hunk 5]
```diff
@@ -39,37 +39,37 @@ We respect the time and talent that drives new discoveries in web security techn
...
 * Adino Namchu
-* [Atulkumar Hariba Shedage](http://securitysolution.co.in)
+* Atulkumar Hariba Shedage
```
Comment (claudiob) [inline security/response/index.md:64]: Snap 2020-08-06 at 2 07 45 PM

File: security/response/index.md [hunk 6]
```diff
@@ -39,37 +39,37 @@ We respect the time and talent that drives new discoveries in web security techn
...
 * [West Arete](http://westarete.com)
 * [Abhinav Karnawat \/ w4rri0r \/](http://www.w4rri0r.com)
 * [Mahadev subedi](https://twitter.com/blinkms)
 * [Vedachala](https://twitter.com/vedachalaka)
 * [Ehraz Ahmed](https://ehraz.co)
 * [Umraz Ahmed](https://twitter.com/umrazahmed)
 * [Ahsan Akhtar](http://www.peopleperhour.com/freelancer/ahsan/website-web-application-security/335173)
-* [Jose Pino](https://twitter.com/Fr4phc0r3)
+* Jose Pino
```
Comment (claudiob) [inline security/response/index.md:72]: Snap 2020-08-06 at 2 07 50 PM

File: security/response/index.md [hunk 7]
```diff
@@ -78,35 +78,35 @@ We respect the time and talent that drives new discoveries in web security techn
 * [Hammad Shamsi](https://www.facebook.com/NiNJA.Sh3iFU)
 * [Saurabh Chandrakant Nemade](https://www.facebook.com/saurabh.nemade)
 * [Nitin Goplani](https://www.linkedin.com/in/nitingoplani)
-* [Sahil Saif](https://twitter.com/bewithsahilsaif)
+* Sahil Saif
```
Comment (claudiob) [inline security/response/index.md:81]: Snap 2020-08-06 at 2 07 54 PM

File: security/response/index.md [hunk 8]
```diff
@@ -78,35 +78,35 @@ We respect the time and talent that drives new discoveries in web security techn
...
 * [Rafael Pablos](http://silverneox.blogspot.com)
 * Nutan Kumar Panda
 * [Koutrouss Naddara](https://www.facebook.com/superbade)
 * [Gurjant Singh](https://twitter.com/GurjantSadhra)
 * [Mayank Kapoor](https://twitter.com/wHys0SerI0s)
-* [FailHunters Crew](http://www.failhunters.com)
+* FailHunters Crew
```
Comment (claudiob) [inline security/response/index.md:87]: Snap 2020-08-06 at 2 07 57 PM

File: security/response/index.md [hunk 9]
```diff
@@ -78,35 +78,35 @@ We respect the time and talent that drives new discoveries in web security techn
...
 * [Daniel Alvear(MaztoR IN-Security)](https://twitter.com/mazt0r)
 * [Ali Hassan Ghori (@alihasanghauri)](http://alihassanpenetrationtester.blogspot.com/)
 * [Rodolfo Godalle Jr.](https://www.facebook.com/junior.ns1de)
 * [Simone Memoli](https://twitter.com/Simon90_Italy)
 * [Daksh Patel](https://www.facebook.com/dakshxss)
 * [Jovan Šikanja](http://www.e-sigurnost.net)
 * [Muhammad Talha Khan](https://www.facebook.com/MTK911)
-* [Yash Pandya](https://twitter.com/eryash9_yash)
+* Yash Pandya
```
Comment (claudiob) [inline security/response/index.md:95]: Snap 2020-08-06 at 2 08 02 PM

File: security/response/index.md [hunk 10]
```diff
@@ -78,35 +78,35 @@ We respect the time and talent that drives new discoveries in web security techn
...
 * [Mark Dodwell](https://twitter.com/madeofcode)
 * [Gabe Marshall](http://www.gabemarshall.me)
 * [Matt Jaynes](http://mattjaynes.com/)
 * [Nakul Mohan (@Anonymous\_India)](https://www.facebook.com/nakul.cia)
 * [Hardik Tailor](https://twitter.com/iamhardiktailor)
-* [Marques Johansson](https://plus.google.com/+MarquesJohansson)
+* Marques Johansson
```
Comment (claudiob) [inline security/response/index.md:101]: Snap 2020-08-06 at 2 08 07 PM

File: security/response/index.md [hunk 11]
```diff
@@ -78,35 +78,35 @@ We respect the time and talent that drives new discoveries in web security techn
...
 * [Rishiraj Sharma](https://twitter.com/ehrishiraj)
 * [Yogendra Sharma](https://twitter.com/FuzzBaBa)
 * [Prashant Padmashali](https://www.linkedin.com/in/prashantpadmashali/)
-* [Apoorv Joshi](https://twitter.com/apo143u)
+* Apoorv Joshi
```
Comment (claudiob) [inline security/response/index.md:105]: Snap 2020-08-06 at 2 08 13 PM

File: security/response/index.md [hunk 12]
```diff
@@ -78,35 +78,35 @@ We respect the time and talent that drives new discoveries in web security techn
...
 * [Shivam Kumar Agarwal, Nithish Varghese, and Sahil Srivastava](https://twitter.com/netanalysts)
-* [Shahmeer Amir](http://www.maadssec.com)
+* Shahmeer Amir
```
Comment (claudiob) [inline security/response/index.md:107]: Snap 2020-08-06 at 2 08 19 PM

File: security/response/index.md [hunk 13]
```diff
@@ -78,35 +78,35 @@ We respect the time and talent that drives new discoveries in web security techn
...
 * [Hamid Ashraf](https://twitter.com/hamihax)
-* [Babar Khan Akhunzada](http://www.babarkhan.ml)
+* Babar Khan Akhunzada
```
Comment (claudiob) [inline security/response/index.md:109]: Snap 2020-08-06 at 2 08 23 PM

**Comment by kaspth** [comment]
@claudiob hey, long time no see! Thanks for going through all these 🙌😄

**Pull request merged**